spec/lib/diaspora_federation/entities/comment_spec.rb

```diff
@@ -0,0 +1,27 @@
+module DiasporaFederation
+  describe Entities::Comment do
+    let(:data) {
```

Comment:
I started with FactoryGirl in my specs, see https://github.com/SuperTux88/diaspora_federation/blob/salmon/spec/lib/diaspora_federation/entities/profile_spec.rb#L3

I didn't see this in the other Pull-Requests (sorry), but I think it's better when we change it in all specs.